python/lsst/jointcal/utils.py

```diff
@@ -350,23 +349,23 @@ def _make_match_dict(self, reference, visit_catalogs, photoCalibs, refcalib=None
         def get_fluxes(photoCalib, match):
             """Return (flux, ref_flux) or None if either is invalid."""
             # NOTE: Protect against negative fluxes: ignore this match if we find one.
-            maggiesToJansky = 3631
             flux = match[1]['slot_CalibFlux_instFlux']
             if flux < 0:
                 return None
             else:
-                flux = maggiesToJansky * photoCalib.instFluxToMaggies(match[1], "slot_CalibFlux").value
+                flux = photoCalib.instFluxToNanojansky(match[1], "slot_CalibFlux").value
 
-            # NOTE: Have to protect against negative reference fluxes too.
+            # NOTE: Have to protect against negative "reference" fluxes too.
```

Comment:
Off-topic, but can you use a sourceSelector on the reference catalog for a s/n or flux>0 cut?